modules/firebase/examples_test.go

```diff
@@ -0,0 +1,40 @@
+package firebase_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/firebase"
+)
+
+func ExampleRun() {
+	ctx := context.Background()
+
+	firebaseContainer, err := firebase.Run(ctx, "ghcr.io/u-health/docker-firebase-emulator:13.29.2",
+		firebase.WithRoot(filepath.Join("testdata", "firebase")),
```

Comment:
info: using literal "testdata/firebase" works fine on all platforms as even though windows uses `\` it also accepts `/`.